root/static/scripts/tests/Control/URLCleanup.js

```diff
@@ -4962,6 +4973,43 @@ function doMatchSubtest(
   previousMatchTests.push(entityType + '+' + url);
 }
 
+function doOptionsSubtest(
+  st,
+  entityType,
+  url,
+  label,
+  expectedRelationshipOptions,
+) {
+  const checker = new Checker(cleanURL(url), entityType);
+  const possibleOptions = checker.getPossibleTypes();
+  let actualRelationshipOptions = possibleOptions || undefined;
+  if (expectedRelationshipOptions === undefined) {
+    actualRelationshipOptions = undefined;
+  }
+
+  const relationshipOptions = possibleOptions.reduce(function (accum, uuid) {
+    if (relationshipTypesByUuid[uuid]) {
+      accum.push(relationshipTypesByUuid[uuid]);
+    }
+    return accum;
+  }, []);
+
+  if (relationshipOptions.length > 0) {
+    actualRelationshipOptions = relationshipOptions;
+  }
+
+  const msg = 'Match ' + label + ' URL relationship options for ' +
+  entityType + ' entities';
+  st.ok(
+    arraysEqual(
+      actualRelationshipOptions.sort(),
+      expectedRelationshipOptions.sort(),
+    ),
+    msg,
+  );
+  previousOptionsTests.push(entityType + '+' + url);
+}
```

Comment:
Implementing MBS-11980 is probably worth a separate commit, if not a separate pull request.

Reply:
Oh, true, we had a ticket for this. Opened a new PR, https://github.com/metabrainz/musicbrainz-server/pull/2339